client/src/lib/oauth/state-machines/factory.ts

```
@@ -0,0 +1,149 @@
+            "Use 'dcr' or 'preregistered' instead.",
+        );
+      }
+      return create2025_06_18(baseConfig as Config2025_06_18);
+
+    case "2025-11-25":
+      // All registration strategies are valid for 2025-11-25
+      return create2025_11_25(baseConfig as Config2025_11_25);
+
+    default:
+      // TypeScript exhaustiveness check
+      const _exhaustive: never = protocolVersion;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)